crates/core/tedge/src/error.rs

```diff
@@ -30,4 +30,7 @@ pub enum TEdgeError {
 
     #[error(transparent)]
     FromConfigNotSet(#[from] tedge_config::ConfigNotSet),
+
+    #[error("Certificate {0} does not exist to renew")]
+    CertificateDoesNotExist(String),
```

Comment:
There is already an error case defined for that purpose: `CertError::CertificateNotFound`

```suggestion
```

Reply:
removed